### ISSUE TITLE
such a small thing needs another plugin ? why not incorporate it in main theme

### ISSUE THREAD

**Issue by prakash4mail** [tomusborne/amp-for-generatepress#9]
such a small thing needs another plugin ? why not incorporate it in main theme.
it would require maintenance from our end to update.
i would suggest change the default structure of menu that works with both amp & nonAmp Automatically

**Comment by jamesozzie** [comment]
I second this, many users are not aware of it's compatibility.

**Comment by rsmith4321** [comment]
Yes I agree. If they did this they could add the theme here and get a lot of good exposure. https://amp-wp.org/ecosystem/themes/
